### PR TITLE
Redeeming a free hackclub.com subdomain 

### PR DESCRIPTION
**Pull request by LightningFz**

### CODE REVIEW

**Review by grymmy — CHANGES_REQUESTED** [review]
This file is kept in alphabetical order.  Please relocate this block accordingly.

Additionally, please add a good high-level description to your PR stating what you intend this change to actually do.

File: hackclub.com.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ---
+slss.hackclub.com:
+  - ttl: 1
+    type: CNAME
+    value: slss.
```
Comment (Muirrum) [inline hackclub.com.yaml:5]: This DNS record seems non-operative. What domain are you trying to point this to?

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 1 above)
Comment (LightningFz) [reply hackclub.com.yaml:5]: I want the url to be slss.hackclub.com I'm sorry but I'm finding difficulty sitting it up

Comment (Muirrum) [reply hackclub.com.yaml:5]: What do you want slss.hackclub.com to point to? Also please see Graham's comment about providing a description for this PR.

**Comment by reesericci** [comment]
Closing due to inactivity, feel free to re-open once you have addressed the issues outlined!

**Pull request closed without merging**